[Outlook] (metadata) Update ms.topic values (#3939)
OfficeDev/office-js-docs-pr: docs/outlook/web-services.md

> [!IMPORTANT]
> EWS calls and operations aren't supported in add-ins running in Outlook on iOS and Android.

> The way that you call a web service varies based on where the web service is located. The following tables lists the different ways that you can call a web service based on location.

Reviewer: ```suggestion
The way you call a web service varies based on where the web service is located. The following tables list the different ways you can call a web service based on location.
```

> |**Web service location**|**Way to call the web service**|

Reviewer: No need to explicitly bold top row for Learn site

```suggestion
|Web service location|Way to call the web service|
```

The following table lists the EWS operations that add-ins support. To see examples of SOAP requests and responses, choose the link for each operation. For more information about EWS operations, see [EWS operations in Exchange](/exchange/client-developer/web-service-reference/ews-operations-in-exchange).

> |**EWS operation**|**Description**|

Reviewer: ```suggestion
|EWS operation|Description|
```
